winui/Masked-TextBox/Mask-Types.md

```diff
@@ -7,4 +7,192 @@ control: SfMaskedTextBox
+
+{% endhighlight %}
+{% endtabs %}
+
+![Mask type simple in WinUI MaskedTextBox](MaskedTextBox_Images/winui_mask_type_simple.png)
+
+![Mask type simple with value in WinUI MaskedTextBox](MaskedTextBox_Images/winui_simple_mask_with_value.png)
+
+## RegEx
+
+The expressions that are generated with regular expressions come under this group. For example: hexadecimal values [0-9A-C].
+
+### RegEx mask elements
+
+<table>
+<tr>
+<th>
+Elements</th><th>
+Description</th></tr>
+<tr>
+<td>
+[ABC]</td><td>
+Accepts any single character included in the specified set of characters.</td></tr>
+<tr>
+<td>
+[^ABC]</td><td>
+Accepts any single character not included in the specified set of characters.</td></tr> 
+<tr>
+<td>
+[0-9A-Z]</td><td>
+Accepts any digit between 0-9 and character between A-Z.</td></tr>
+<tr>
+<td>
+\d</td><td>
+Accepts any digit. Same as [0-9].</td></tr>
+<tr>
+<td>
+\D</td><td>
+Accepts any non digit. Same as [^0-9].</td></tr>
+<tr>
+<td>
+\w</td><td>
+Accepts any  word character. \w is the same as [a-zA-Z_0-9].</td></tr>
+<tr>
+<td>
+\W</td><td>
+Accepts any  non word character. \W is the same as [^a-zA-Z_0-9].</td></tr>
+<tr>
+<td>
+\s</td><td>
+Accepts any  white space characters.</td></tr>
+<tr>
+<td>
+\S</td><td>
+Accepts any non white space characters.</td></tr>
+<tr>
+<td>
+\S</td><td>
+Accepts any non white space characters.</td></tr>
+<tr>
+<td>
+(?=ABC)</td><td>
+Matches a group after the main expression without including it in the result.</td></tr>
+<tr>
+<td>
+(?!ABC)</td><td>
+Specifies a group that cannot match after the main expression.</td></tr>
+<tr>
+<td>
+{n}</td><td>
+Accepts the input for n number of times.</td></tr>
+<tr>
+<td>
+{n,}</td><td>
+Accepts the input for 'n' and more than 'n' number of times. </td></tr>
+<tr>
+<td>
+{n,m}</td><td>
+Accepts the input for n minimum number of times and m maximum number of times.</td></tr>
+<tr>
+<td>
++</td><td>
+Accepts one or more matches for the preceding character.</td></tr>
+<tr>
+<td>
+*</td><td>
+Accepts zero or more matches for the preceding character.</td></tr>
+<tr>
+<td>
+?</td><td>
+Optional input (Zero or one occurrence  of the matching input)</td></tr>
+<tr>
+<td>
+|</td><td>
+Acts like a Boolean OR. Matches the expression before or after the |.</td></tr>
+<tr>
+<td>
+.</td><td>
+Accepts any character. It can be changed based on culture</td></tr>
+</table>
+
+{% tabs %}
+{% highlight XAML %}
+
+<syncfusion:SfMaskedTextBox x:Name="sfMaskedTextBox"
+                            MaskType="RegEx"
+                            Mask="(0[0-9]|1[1-9]|2[1-3]){2}:[0-5][0-9]:[0-5][0-9]" />
+
+{% endhighlight %}
+{% highlight C# %}
+
+SfMaskedTextBox maskedTextBox = new SfMaskedTextBox();
+maskedTextBox.MaskType = MaskedTextBoxMaskType.RegEx;
+maskedTextBox.Mask = "(0[0-9]|1[1-9]|2[1-3]){2}:[0-5][0-9]:[0-5][0-9]";
+
+{% endhighlight %}
+{% endtabs %}
+
+![Mask type RegEx in WinUI MaskedTextBox](MaskedTextBox_Images/winui_mask_type_regex.png)
```

Comment:
@Vignesh-SF3580 Change the code snippet with the setting Value property and add a single image for both simple and regex.

Reply:
Added gif images for both simple and regex mask.